apps/client/src/app/auth/ui/desktop-login-view/desktop-login-view.component.scss

```diff
@@ -0,0 +1,86 @@
+@use 'variables/colors';
+
+:host {
+  display: flex;
+  height: 100%;
+  background-color: colors.getPrimary(800);
+
+  .pikachu {
+    max-width: 428px;
+    max-height: 428px;
+
+    object-fit: contain;
+    position: inherit !important;
+  }
+
+  .modal {
```

Comment:
Duplicate style between desktop-login-view and mobile-login-view. Create shared style for auth lib